common/pom.xml

```diff
@@ -77,5 +77,10 @@
             <artifactId>org.jacoco.agent</artifactId>
             <classifier>runtime</classifier>
         </dependency>
+        <!--FindBugs dependencies-->
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+        </dependency>
     </dependencies>
 </project>
```

Comment:
Add new line at the end. Not your code, but still better to fix the existing code. :)

Reply:
:) Fixed.